c2corg_api/scripts/migration/documents/books.py

```diff
@@ -0,0 +1,108 @@
+from c2corg_api.models.book import Book, ArchiveBook, BOOK_TYPE
+from c2corg_api.models.document import DocumentLocale, ArchiveDocumentLocale, \
+    DOCUMENT_TYPE
+from c2corg_api.scripts.migration.documents.document import MigrateDocuments
+from c2corg_api.scripts.migration.documents.routes import MigrateRoutes
+
+
+class MigrateBooks(MigrateDocuments):
+
+    def get_name(self):
+        return 'books'
+
+    def get_model_document(self, locales):
+        return DocumentLocale if locales else Book
+
+    def get_model_archive_document(self, locales):
+        return ArchiveDocumentLocale if locales else ArchiveBook
+
+    def get_document_geometry(self, document_in, version):
```

Comment:
It looks like this is also a problem with the articles, see https://github.com/c2corg/v6_api/issues/456.

We should make sure that no empty geometries are created for books and articles during the migration.